sentinel/db.js

```diff
@@ -17,11 +17,15 @@ if (couchUrl) {
         db: dbName,
         ddoc: ddocName
     };
+    var sentinelAuth = '';
     if (parsedUrl.auth) {
         var index = parsedUrl.auth.indexOf(':');
         module.exports.settings.username = parsedUrl.auth.substring(0, index);
         module.exports.settings.password = parsedUrl.auth.substring(index + 1);
+        sentinelAuth = `${parsedUrl.auth}@`;
     }
+    var sentinelUrl = `${parsedUrl.protocol}//${sentinelAuth}${parsedUrl.host}/sentinel`;
```

Comment:
I think the db name should be `dbName + '-sentinel'` just like the auditName on line 10.